elasticsearch-agent/src/main/java/com/epam/pipeline/elasticsearchagent/service/impl/converter/pipeline/PipelineCodeMapper.java

```diff
@@ -42,7 +46,7 @@ public XContentBuilder pipelineCodeToDocument(final Pipeline pipeline,
                     .field("pipelineName", pipeline.getName())
                     .field("pipelineVersion", pipelineVersion)
                     .field("path", path)
-                    .field("content", fileContent);
+                    .field("content", buildFileContent(fileContent));
```

Comment:
Please, check whether it is possible to limit actual downloaded content from Gitlab (not in our API, but in Gitlab API), in case of really large files we'll fail with OOM at the load time.